include/drivers/sensor.h

```diff
@@ -174,6 +174,24 @@ enum sensor_channel {
 	/** Desired charging current in mA */
 	SENSOR_CHAN_GAUGE_DESIRED_CHARGING_CURRENT,
 
+	/** CH0 Capacitance, in Picofarad **/
+	SENSOR_CHAN_CAPACITANCE_CH0,
+	/** CH1 Capacitance, in Picofarad **/
+	SENSOR_CHAN_CAPACITANCE_CH1,
+	/** CH2 Capacitance, in Picofarad **/
+	SENSOR_CHAN_CAPACITANCE_CH2,
+	/** CH3 Capacitance, in Picofarad **/
+	SENSOR_CHAN_CAPACITANCE_CH3,
+
+	/** CH0 Frequency, in MHz **/
```

Comment:
I'm not sure MHz is an ideal unit.  Hz would allow for frequencies like 0.5 Hz, i.e. every two seconds; or 0.000012 (a very inaccurate once-per-day).

This would limit the frequency to 2 GHz, though, which might be limiting in some applications.  Perhaps a future sensor API could select the best unit based on the value, and return either frequency in Hz or interval in some time unit, depending on which is more accurate.

This could be discussed in the API meeting.  Sticking with MHz or even kHz would be a reasonable resolution of such a discussion.

Reply:
I think you're right. Hz would make more sense. I'm not sure if >2GHz would be necessary for sensors, but who knows. Would it be possible to have both units, Hz now and MHz later if needed? This way you could use Hz for <2GHz and MHz for values above. 
Also I agree with your other suggestion. Having the flexibility to get the value returned in the most accurate unit would be really nice.

I will wait for the resolution of the API meeting though and then change it based on that. Could you give me an update on the resolution?

Reply:
How about converting these sensor channel enums to be private to the FDC2X1X driver instead? Similar to how it is done for the MCUX ACMP sensor driver in https://github.com/zephyrproject-rtos/zephyr/blob/f71cd2aaf5e3f5942484a7ddb92e6192b2130dc6/include/drivers/sensor/mcux_acmp.h#L21?

Reply:
API meeting 26th Jan 2021:

Moving the enums to the driver-specific header is the concluded best approach for this.

Reply:
Thank you both for your suggestion!  I moved the channel enums to the driver-specific header now